tests/conftests/platforms.py

```diff
@@ -0,0 +1,166 @@
+"""
+In this we keep the platforms specify test configuration.
+The file has been strucutred as fllows:
+    - platform
+    - non_platform
+    - pool platform
+    - pool non_platform
```

Comment:
only nit pick: 
I did have to look up in the code what you mean with the term "pool". 

Idea is good, I'm fine with the term now, but maybe `platform set`  or `platform exclusion set` is more self explanatory